.github/workflows/checks.yaml

```diff
@@ -61,14 +61,21 @@ jobs:
 
   validate-publish:
     runs-on: ubuntu-latest
-    # only run on release pull requests
-    if: ${{ startsWith(github.ref, 'refs/head/release') && contains(github.event.sender, 'rmconsole') }}
```

Comment:
This wasn't working, opting for a different approach, calculating the release branchname using the current pubspec.yaml